### PR TITLE
update agent config to upload firefly

### PR DESCRIPTION
**Pull request by mol-george**
This PR implements [VC-30108](https://venafi.atlassian.net/browse/VC-30108) by updating the tlspk agent configuration to gather and update firefly.

Related to MR [#168](https://gitlab.com/venafi/vaas/applications/tls-protect-for-k8s/venctl/-/merge_requests/168/edit)

[VC-30108]: https://venafi.atlassian.net/browse/VC-30108?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: deploy/charts/venafi-kubernetes-agent/templates/configmap.yaml [hunk 1]
```diff
@@ -191,4 +191,11 @@ data:
           group: jetstack.io
           version: v1alpha1
           resource: venafiissuers
+    - kind: "k8s-dynamic"
+      name: "k8s/fireflyissuers"
+      config:
+        resource-type:
+          group: firefly.venafi.com
+          version: v1
+          resource: firefly
```
Comment (tfadeyi) [inline deploy/charts/venafi-kubernetes-agent/templates/configmap.yaml:200]: Suggestion(blocking): The resource is `issuer`

File: deploy/charts/venafi-kubernetes-agent/templates/configmap.yaml [hunk 2]
```diff
@@ -191,4 +191,11 @@ data:
...
           resource: venafiissuers
-{{- end }}
+    - kind: "k8s-dynamic"
+      name: "k8s/fireflyissuers"
+      config:
+        resource-type:
+          group: firefly.venafi.com
+          version: v1
+          resource: fireflyissuers
+{{- ed }}
```
[the remark this answers is not in the document]
Comment (mol-george) [reply deploy/charts/venafi-kubernetes-agent/templates/configmap.yaml]: fixed af28dfa973a79a893f96886ae39f0dc2eab7b7bc

File: deploy/charts/venafi-kubernetes-agent/templates/configmap.yaml (hunk 1 above)
Comment (mol-george) [reply deploy/charts/venafi-kubernetes-agent/templates/configmap.yaml]: Changed it to `issuers` - **plural**.

**Review by tfadeyi — APPROVED** [review]
LGTM

**Pull request merged**